pkg/sloop/webfiles/sloop_ui.js

```diff
@@ -257,6 +260,7 @@ function bindMouseEvents(svg) {
                     detailedToolTipIsVisible = true;
                     tooltip.html(result);
                     evalJSFromHtml(result);
+                    positionTooltip(tooltipX, tooltipY)
```

Comment:
is this going to cause a flicker (visible for a short time in original location/size, then move/resize based on additional data)?

Reply:
not sure I understand the need for the `else` half of this if